users/ritvikkolhe/Task 3/Quiz.html

```diff
@@ -0,0 +1,52 @@
+        <br><br><h2>Welcome to IEEE Quiz</h2><br><br>
+        <h4>Q.1 Which of the following is the best club/chapter in VIT ?</h4>
+        <input type="radio" name="" onclick="add()" value="Correct"> IEEE-VIT<br>
+        <input type="radio" name="" value="Incorrect"> CSI<br>
+        <input type="radio" name="" value="Incorrect"> GDG<br>
+        <input type="radio" name="" value="Incorrect"> ACM<br><br>
+        <h4>Q.2 Total number of people in IEEE-VIT family ?</h4>
+        <input type="radio" name="" value="Incorrect"> 70<br>
+        <input type="radio" name="" value="Incorrect"> 71<br>
+        <input type="radio" name="" value="Incorrect"> 75<br>
+        <input type="radio" name="" onclick="add()" value="Correct"> 74<br><br>
+        <h4>Q.3 Who is the Tech Head (Web) in IEEE-VIT Chapter ?</h4>
+        <input type="radio" name="" value="Incorrect"> Karishnu Poddar<br>
+        <input type="radio" name="" onclick="add()" value="Correct"> Arushit Mudgal<br>
+        <input type="radio" name="" value="Incorrect"> Mayur Shastri<br>
+        <input type="radio" name="" value="Incorrect"> Udbhav Tripathi<br><br>
```

Comment:
You need to give same `name` to the same set of options, otherwise a person can select all the options.